README.md

```diff
@@ -236,7 +236,7 @@ print(f'PSNR = {psnr.numpy():3f}')
 trainer.model.save_weights('weights/wdsr-b-32-x4/weights.h5')
 ```
 
-## SRGAN
+### SRGAN
 
 ### Generator pre-training
```

Comment:
Should start with `####` then? Also next section ...

Reply:
Shouldn't what? SRGAN? Is it a subset of WDSR (above it) or Pre-trained weights?

Reply:
Sections `Generator pre-training` and `Generator fine-tuning (GAN)` are subsections of `SRGAN`, so when you increase the level of `SRGAN` from `##` to `###` the levels of these subsections should be increased from `###` to `####`.

Reply:
Ok, I follow now! Updated.